test/unit_tests/util/PathTest.cpp

```diff
@@ -55,65 +55,65 @@ TEST(UtilPath, testClearExtensions) {
     auto a = fs::path("C:") / "test" / "abc" / "xyz.txt";
     fs::path old_path(a);
     Util::clearExtensions(a);
-    EXPECT_EQ(old_path.string(), a.string());
+    EXPECT_EQ(old_path.u8string(), a.u8string());
 
     a = fs::path("C:") / "test" / "abc" / "xyz";
     old_path = a;
     a += ".xopp";
     Util::clearExtensions(a);
-    EXPECT_EQ(old_path.string(), a.string());
+    EXPECT_EQ(old_path.u8string(), a.u8string());
 
 
     // The following tests use the generic separator which works on all systems
     auto b = fs::path("/test/asdf.TXT");
     Util::clearExtensions(b);
-    EXPECT_EQ(string("/test/asdf.TXT"), b.string());
+    EXPECT_EQ(string("/test/asdf.TXT"), b.u8string());
```

Comment:
Either the literal must be also in u8 or b.string() must be used.

```suggestion
    EXPECT_EQ(string(u8"/test/asdf.TXT"), b.u8string());
```

This applies for all assertions.